spyder/plugins/projects/widgets/qcookiecutter.py

```diff
@@ -0,0 +1,420 @@
+# -*- coding: utf-8 -*-
+# -----------------------------------------------------------------------------
+# Copyright © Spyder Project Contributors
+#
+# Licensed under the terms of the MIT License
+# (see spyder/__init__.py for details)
+# -----------------------------------------------------------------------------
+"""

```

Comment:
Please add a blank line above this line.

Reply:
Fixed